lib/unobtainium/drivers/selenium.rb

```diff
@@ -55,9 +55,29 @@ def ensure_preconditions(_, _)
                 err.backtrace
         end
 
+        def clean_chrome_args(options)
+          options["desired_capabilities"]["chromeOptions"]["args"].uniq!
```

Comment:
Personally I think you should not do the destructive approach here for 2 main reasons. First you are returning a value, so the caller already expects to use the return value. Second, you destroy the argument, meaning you're going to mutate the argument. This is a side effect that the caller might not expect. This might lead to some bugs where someone calls this method and keeps using the argument thinking it's still like it was before.

I assume you use the return value of ``clean_chrome_args`` so couldn't we write it like:

```ruby
def clean_chrome_args(options)
   options["desired_capabilities"]["chromeOptions"]["args"].uniq
rescue
   options
end
```
The main difference here is that we're not mutating the passed argument and once I call ``new_options = clean_chrome_args(old_options)`` I can be sure that ``old_options`` stay the same.

If in the other hand you really really really need to destroy the array :P maybe you should add a ``!`` to the method name - ``clean_chrome_args!``. This is the convention in ruby right?